### PR TITLE
Take your account with you UI changes

### PR DESCRIPTION
**Pull request by ewjordan**
Item 1 in https://docs.google.com/document/d/1R8oBFcixpN1X_FQJq9G5-l3AbaqjGSqvD9oDwL00Wns/edit#heading=h.ezfouxcqg3na

(Update) New text for enter_parent_email: "Enter your parent or guardian's email address (for password recovery)"

Personal word login (with email):
personal-login-words

Personal word login (no email - note, parenthetical "to recover password" is now removed, though it was not when I took this screenshot):
personal-login-words-no-email

Personal picture login:
personal-login-picture

Delete account button:
delete-account

### CODE REVIEW

**Comment by ewjordan** [comment]
@poorvasingal please take a look at the images, see if these are what you had in mind. I reordered some elements the way the doc seemed to suggest.

**Comment by breville** [comment]
Oh, is there any testing of this UI?  Seems like it would be great to have some.

**Comment by poorvasingal** [comment]
Cool, looks good. Just a couple of comments: 
- Can we actually say "Enter in your parent or guardian's email address (for password recovery)" instead? I realized that the way I had it in the spec makes it sound like we're saying you want to recover your password right now. 
- Why are we changing the note for Deleting accounts to include the fact that this does not delete student accounts? I don't think we should do that since we're planning on changing that behavior soon.

**Comment by breville** [comment]
Suggestion: "Enter your parent or..." rather than "Enter in your parent or..."

**Comment by ewjordan** [comment]
@poorvasingal, the deleting accounts text is that way right now in production, that's not part of this changeset (this change just makes the header teal). Would you like me to change that to something else?

**Comment by ewjordan** [comment]
(Spoke with Poorva, we'll leave the delete text alone for now)

File: dashboard/app/views/devise/registrations/edit.html.haml [hunk 1]
```diff
@@ -86,40 +86,44 @@
...
-        = form.label :secret_words
-        = form.text_field :secret_words, autocomplete: 'off', value: '', maxlength: 255
     .field
       = form.label :password, maxlength: 255
       = form.password_field :password, autocomplete: 'off', maxlength: 255
     .field
       = form.label :password_confirmation
       = form.password_field :password_confirmation, autocomplete: 'off', maxlength: 255
+    - if current_user.secret_word_account?
+      %h3= t('user.confirm_secred_words')
```
Comment (ewjordan) [inline dashboard/app/views/devise/registrations/edit.html.haml:118]: Grr, I typoed this by accident, I don't know how I didn't catch that. Fixing in another PR

**Pull request merged**